src/components/common/slider-range/slider-range-track.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { Ref, computed, ref } from 'vue'
```

Comment:
shouldn't it be called "active-track" or "track-selection" ?
If I understand correctly this component represents the highlighted zone between the thumb handles ?

Reply:
Good idea, renamed `slider-range-track` to `slider-range-active-track`